.changeset/heavy-buckets-try.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-t3-app": patch
+---
+
+upgrade prettier and eslint (for docs)
```

Comment:
rm this.

we're not updating any cli logic or files for the scaffolded app, so a publish here won't do anything. it's just devDeps being updated